### ISSUE TITLE
generate duplicate fields in schema for mssql

### ISSUE THREAD

**Issue by chiyoung0258** [confluentinc/kafka-connect-jdbc#42]
when I use jdbc connect for mssql, schema has duplicate fields. like this;
```
{"schema":{"type":"struct", "fields":[{"type":"int8", "optional":false, "field":"paidFlag"}, {"type":"int16", "optional":false, "field":"portNo"}, {"type":"int16", "optional":false, "field":"portNo"}, 
{"type":"int16", "optional":false, "field":"portNo"}, .... ], "optional":false, "name":"MyServer"}, "payload":{ ...}
```
I found duplicate "portNo" fields in result schema.
I tried to connect another table, I found same problems in schema, but payload values are correct.

How can I resolve it? 
Thanks.

**Comment by chiyoung0258** [comment]
confluent kafka-0.9-cp1
hadoop-2.7.1
sqljdbc4.jar (from https://www.microsoft.com/ko-kr/download/details.aspx?id=11774)

this is my environments.

**Comment by ewencp** [comment]
@chiyoung0258 Are you using `JsonConverter` to store the data or `AvroConverter`? This sounds like https://issues.apache.org/jira/browse/KAFKA-3055, which is fixed and will be released in Kafka 0.9.0.1.

**Comment by chiyoung0258** [comment]
@ewencp 
I built Kafka 0.9.1 and exchange "share/java/kafka/connect-json-0.9.0.0-cp1.jar" for "share/java/kafka/connect-json-0.9.1.0-SNAPSHOT.jar".
And then, it works fine. 
thanks for your suggestion.
